core/src/main/java/com/novoda/noplayer/player/PlayerFactory.java

```diff
@@ -14,6 +14,8 @@
 
 public class PlayerFactory {
 
+    private static final boolean USE_SECURE_CODEC = false;
```

Comment:
Yes, they can. We need this for the `intel` hack. Previously we did the hack in the library but it isn't really it's responsibility to know the hack, it relies on the content security which the library does not know about.